content/cucumber/step-definitions.md

```diff
@@ -165,7 +181,7 @@ So if you try `--guess` with the mice above, Cucumber will pick `/Three blind (.
 
 ## Redundant Step Definitions
 
-In Cucumber, you're not allowed to use a `Regexp` more than once in a Step Definition—even across files, and even with different code inside the Proc. 
+In Cucumber, you're not allowed to use a `Regexp` more than once in a Step Definition—even across files, and even with different code inside the method or function.
```

Comment:
Is this `Regexp`  specific cucumber use of regular expression? if not we should just use Regular Expression

Reply:
I'm not actually sure... The use of "Regexp' was fairly consistent throughout the docs, so I figured I'd keep it like that. I did mention "Regular Expressions (`Regexp`)" somewhere near the top of the docs. Could be updated separately if we want?